apps/spa/src/environments/environment.template

```diff
@@ -2,20 +2,5 @@ export const environment = {
 	production: $IS_PRODUCTION,
 	deploymentName: '$DEPLOYMENT_NAME',
 	apiUrl: '$API_URL',
-	oauth: { // todo: replace this with /$/OAUTH_CONFIG before merge into main
-         		config: {
-         			redirectUri: window.origin + '/auth',
-         			oidc: true,
-         			responseType: 'code',
-         			clientId: '6b5aa2b3-6237-44ba-8448-252052e73831',
-         			issuer:
-         				'https://kordisleitstelle.b2clogin.com/5b974891-a530-4e68-ac04-e26a18c3bd46/v2.0/',
-         			tokenEndpoint:
-         				'https://kordisleitstelle.b2clogin.com/kordisleitstelle.onmicrosoft.com/b2c_1_signin/oauth2/v2.0/token',
-         			scope: 'openid offline_access 6b5aa2b3-6237-44ba-8448-252052e73831',
-         			strictDiscoveryDocumentValidation: false,
-         		},
-         		discoveryDocumentUrl:
-         			'https://kordisleitstelle.b2clogin.com/kordisleitstelle.onmicrosoft.com/B2C_1_SignIn/v2.0/.well-known/openid-configuration',
-         	},
+	oauth: $OAUTH_CONFIG as any,
```

Comment:
```suggestion
	oauth: $OAUTH_CONFIG as unknown,
```

Reply:
Sadly, with unknown the angular compiler isn't happy :( since it is not effecting any types during development, i think _any_ might be acceptable

Reply:
Right, because unchecked access in file apps/spa/src/app/app.module.ts in lines 34 and 35 does not work. Why do we have to cast the value in the first place?

Reply:
are we ok with leaving this to `any`? Typing is needed because we have strict compiler settings and if we do not type the environment to have an optional `oauth` property, it will complain about it.

Reply:
I resolved the problem and removed the workaround with 2b13da5. The issue with `any` would have been that only shallow object access would have been possible, but no deep one.